utils/data-processing-family-task-submitter/nbactions.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think this file should be in here... It's just some Netbeans project file, right? Update gitignore.